File: terraMarket/src/main/java/com/example/demo/repository/entity/Mercado.java
package com.example.demo.repository.entity;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import lombok.Data;

@Data
@Entity
@Table(name = "mercados")
public class Mercado {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;
    private String nombre;
    @Column(name = "direccion_f")
    private String direccionF;
    private String telefono;
    private String email;
    private String imagen;
    private boolean activo;

}

File: Front/terraMarket/src/app/models/mercado.ts
import { Direccion } from './direccion';

export interface Mercado {
  id: number;
  nombre: string;
  email: string;
  direccionF: string;
  telefono: string;
  activo: boolean;
  imagen: string;
  direccion: Direccion;
  listaTiendas: any[];
}
